ballerina/tests/tests.bal

```diff
@@ -28,6 +28,8 @@ configurable string accountSid = isTestOnLiveServer ? os:getEnv("ACCOUNT_SID"):
 configurable string authToken = isTestOnLiveServer ? os:getEnv("AUTH_TOKEN"): "AU12345678901234567890123456789012";
 configurable string toPhoneNumber = isTestOnLiveServer ? os:getEnv("TO_PHONE"): "+011234567890";
 configurable string fromPhoneNumber = isTestOnLiveServer ? os:getEnv("TWILIO_PHONE"): "+098765432101";
+configurable string apiKey = isTestOnLiveServer ? os:getEnv("API_KEY"): "SCdegjk35502ga02bbr5311h909a02t3x5";
```

Comment:
Why have not we added test cases using this approach. Better to add at least one test case using this approach.

Reply:
Since this is a draft PR I still haven't pushed those yet. I will add them shortly..

Reply:
Lets update the docs and examples as discussed.